docs/src/catalyst_applications/ode_simulation_performance.md

```diff
@@ -0,0 +1,318 @@
+# [Advice for performant ODE simulations](@id ode_simulation_performance)
+We have previously described how to perform ODE simulations of *chemical reaction network* (CRN) models. These simulations are typically fast and require little additional consideration. However, when a model is simulated many times (e.g. as a part of solving an inverse problem), or is very large, simulation runtimes may become noticeable. Here we will give some advice on how to improve performance for these cases.
+
+Generally, there are few good ways to, before a simulation, determine the best options. Hence, while we below provide several options, if you face an application for which reducing runtime is critical (e.g. if you need to simulate the same ODE many times), it might be required to manually trial these various options to see which yields the best performance ([BenchmarkTools.jl's](https://github.com/JuliaCI/BenchmarkTools.jl) `@btime` macro is useful for this purpose). It should be noted that most default options typically perform well, and it is primarily for large models where investigating these options is worthwhile. All ODE simulations of Catalyst models are performed using the OrdinaryDiffEq.jl package, [which documentation](https://docs.sciml.ai/DiffEqDocs/stable/) provides additional advice on performance.
+
+Generally, this small checklist provides a quick guide for dealing with ODE performance:
+1. If performance is not critical, use [the default solver choice](@ref ode_simulation_performance_solvers) and do not worry further about the issue.
+2. Determine whether your problem is [non-stiff nor stiff](@ref ode_simulation_performance_stiffness), and use the `Tsit5` solver for the former case and `Rodas5P` for the latter.
```

Comment:
Definitely no. Absolutely not. This is not a good idea, and the docs do not recommend this.

Reply:
This is almost certainly worse than the default solver choice. This "step" should just be removed. Either someone should read the solver selection recommendations or they should use the default method. Most likely, they should use the default method.

Reply:
step 2 removed